changelog.d/13800.misc

```diff
@@ -0,0 +1 @@
+Add support for persisting initial room creation events to the DB in a batch.
```

Comment:
Also, I think this should be something like: "Speed up creation of DM rooms."? As that is what readers of the changelog actually care about.

We should probably also merge the base PR changelog into this one?

Reply:
Is there anything else I need to do to merge the base PR changelog into this one other than changing it to be the same?

Reply:
Nope, that's all that is needed!